dev/src/dev/deps_graph.clj

```diff
@@ -56,3 +56,50 @@
   "All namespaces from a module that are used outside that module."
   [module-symb]
   (into (sorted-set) (map :depends-on-namespace) (external-usages module-symb)))
+
+(defn module-dependencies
```

Comment:
Some extra new dev tools to help me find circular references between modules

Reply:
Gave [some feedback](https://github.com/metabase/metabase/pull/42491#discussion_r1596435592) on this in the other PR that used it, in case you were planning on merging this one first.